src/main/java/com/algorand/algosdk/builder/transaction/AssetCreateTransactionBuilder.java

```diff
@@ -169,11 +169,14 @@ public T assetName(String assetName) {
     }
 
     /**
-     * Set url. This value must be between 0 and 32 characters (inclusive).
+     * Set url. This value must be between 0 and 96 characters (inclusive).
      * @param url The asset url.
      * @return this builder.
      */
     public T url(String url) {
+        if (url.length() > 32) {
+            throw new IllegalArgumentException("url must be between 0 and 32 characters (inclusive).");
```

Comment:
```suggestion
            throw new IllegalArgumentException("url must be between 0 and 96 characters (inclusive).");
```

Reply:
I have another PR for this, is it cool if I keep these 32 on this PR?